ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -336,6 +337,7 @@ library
                        -Widentities
                        -Wredundant-constraints
                        -Wmissing-export-lists
+                       -Wno-unticked-promoted-constructors
```

Comment:
Is it ok to ignore this warning?

Reply:
It seems this is the way it should be :tm: https://gitlab.haskell.org/ghc/ghc/-/issues/20531

Reply:
One piece of evidence: This flag was actually removed from `-Wall` in [GHC 9.4](https://downloads.haskell.org/ghc/latest/docs/users_guide/9.4.1-notes.html):

> The [`-Wunticked-promoted-constructors`](https://downloads.haskell.org/ghc/9.4.1/docs/users_guide/using-warnings.html#ghc-flag--Wunticked-promoted-constructors) warning is no longer enabled with [`-Wall`](https://downloads.haskell.org/ghc/9.4.1/docs/users_guide/using-warnings.html#ghc-flag--Wall) ([#20531](https://gitlab.haskell.org/ghc/ghc/issues/20531)), as a part of long-term push towards Dependent Haskell.